### PR TITLE
YLP-2015 Replace loaders by MUI CircularProgress

### PR DESCRIPTION
**Pull request by celia-rouis**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [ ] End-to-end tests successful on CI

### CODE REVIEW

**Review by TimCanu — APPROVED** [review]

File: packages/blip/app/components/chart/daily.js [hunk 1]
```diff
@@ -23,8 +23,7 @@ import WindowSizeListener from 'react-window-size-listener'
 import i18next from 'i18next'
 
 import CalendarTodayIcon from '@mui/icons-material/CalendarToday'
-import InputAdornment from '@mui/material/InputAdornment'
-import TextField from '@mui/material/TextField'
+import { CircularProgress, InputAdornment, TextField } from '@mui/material'
```
Comment (fatb38) [inline packages/blip/app/components/chart/daily.js:26]: Wrong imports, need to import each component
```suggestion
import CircularProgress from '@mui/material/CircularProgress'
import InputAdornment from '@mui/material/InputAdornment'
import TextField from '@mui/material/TextField'
```

File: packages/blip/app/components/chart/trends.js [hunk 2]
```diff
@@ -22,8 +22,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 
 import DateRangeIcon from '@mui/icons-material/DateRange'
-import InputAdornment from '@mui/material/InputAdornment'
-import TextField from '@mui/material/TextField'
+import { CircularProgress, InputAdornment, TextField } from '@mui/material'
```
Comment (fatb38) [inline packages/blip/app/components/chart/trends.js:25]: Same as above

**Pull request merged**